staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -56,6 +59,9 @@ func HTTPWrappersForConfig(config *Config, rt http.RoundTripper) (http.RoundTrip
 		len(config.Impersonate.Extra) > 0 {
 		rt = NewImpersonatingRoundTripper(config.Impersonate, rt)
 	}
+	if len(config.CacheDir) > 0 {
+		rt = NewCacheRoundTripper(config.CacheDir, rt)
```

Comment:
Looks like this also results in all get/list resources getting persisted to disk:

```sh
$ kubectl get secrets

NAME                  TYPE                                  DATA      AGE
default-token-gqj6t   kubernetes.io/service-account-token   3         13m

$ more .kube/http-cache/499e4e6a4f2a2b827ce93a2349098106

HTTP/2.0 200 OK
Content-Length: 3680
Content-Type: application/json
Date: Mon, 07 Aug 2017 00:54:25 GMT

{"kind":"SecretList","apiVersion":"v1","metadata":{"selfLink":"/api/v1/namespaces/default/secrets","resourceVersion":"1105"},"items":[{"metadata":{"name":"default-token-gqj6t"...